### ISSUE TITLE
Fix Storybook accessibility order

### ISSUE THREAD

**Issue by shilman** [storybookjs/storybook#31076]
> I recently started using Storybooks and quite like it but I have a few bits of feedback from an accessibility perspective:
> For whatever reason although the sidebar visually appears on the left (confirmed by my wife). From a screen reader perspective it seems to come after the body of the page. I assume that although visually it has been moved in the actual page code it appears after the main page body.

Originally discussed in https://github.com/storybookjs/storybook/discussions/31042

**Comment by ohnyo64** [comment]
Hello! Would I be able to be assigned to this?

**Comment by S-Maitland** [comment]
@shilman When you are saying that it comes after the body of the page, do you mean it comes at the end of body? when I run storybook it comes at the top of body

**Comment by paulGeoghegan** [comment]
Hi there, I can't speak to the actual HTML order. That was just
speculation. For screen readers you come across the main page content
before the TOC and then last is the actual Storybooks Navigation. In
reality it should be Storybooks nav, TOC, and then the actual page contents.

**Comment by shilman** [comment]
@S-Maitland I guess in screen reader order. I'm just relaying what was posted in the discussion.

**Comment by joonhoswe** [comment]
Hi, my team and I would like to work on this! May I be assigned?

**Comment by shilman** [comment]
No assignment necessary. Please feel free to create a PR and reference the issue in the description!
